vulcanai/tests/models/test_layers.py

```diff
@@ -180,7 +180,8 @@ def test_dense_selu_weight_change(self, dense_unit):
         dense_unit._init_weights()
         new_weight = dense_unit._kernel.weight
         assert (torch.equal(starting_weight, new_weight) is False)
-        assert pytest.approx(round(new_weight.std().item(), 1), 0.1) == std
+        assert (math.isclose(new_weight.std().item(), math.sqrt(1. / fan_in), rel_tol=0.1) is True)
+        #assert pytest.approx(round(new_weight.std().item(), 1), 0.1) == std
```

Comment:
Do you still require this commented line or is it no longer useful?

Reply:
Sorry, I forgot to remove. It is gone now